x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/privilege_monitoring/data_sources/bulk/upsert.ts

```diff
@@ -68,6 +72,7 @@ export const bulkUpsertOperationsFactory =
         ops.push(
           { index: { _index: userIndexName } },
           {
+            '@timestamp': now,
```

Comment:
Is this the only place where an ingest pipeline created the `event.ingested` field?
If that is the case, the ingest pipeline feels quite unnecessary...